twine-upload.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+TWINE_USERNAME=$INPUT_USER \
+  TWINE_PASSWORD=$INPUT_PASSWORD \
```

Comment:
Why did you choose this indentation style? It looks weird. I'd probably avoid indenting env vars but would only indent to command invocation.
WDYT?